scene/3d/skeleton_3d.cpp

```diff
@@ -339,11 +337,14 @@ void Skeleton3D::_notification(int p_what) {
 			thread_local LocalVector<bool> bone_global_pose_dirty_backup;
 
 			// Process modifiers.
+
+			thread_local LocalVector<BonePoseBackup> bones_backup;
 			_find_modifiers();
 			if (!modifiers.is_empty()) {
+				bones_backup.resize(bones.size());
 				// Store unmodified bone poses.
 				for (uint32_t i = 0; i < bones.size(); i++) {
-					bones_backup[i].save(bones[i]);
+					bones_backup.ptr()[i].save(bonesptr[i]);
```

Comment:
```suggestion
					bones_backup[i].save(bonesptr[i]);
```
Or fetch the pointer once above after the `resize`